functions.sh

```diff
@@ -19,6 +19,9 @@ function get_arch() {
 	s390x)
 		arch="s390x"
 		;;
+	arm64)
+		arm64="arm64"
```

Comment:
This should be changed to:

```
arch="arm64"
```

See the shellcheck failure here:

https://travis-ci.org/nodejs/docker-node/jobs/260482790#L486